src/apps/editor/components/matrix/form-renderer.tsx

```diff
@@ -75,31 +77,51 @@ export const FormRenderer: React.FC<Props> = ({
 
   return (
     <VCol variant="md" className={twMerge('w-full', depth > 0 && 'pl-4')}>
-      <Input
-        variant="plain"
-        defaultValue={value}
-        placeholder={type === 'array' ? 'Array key' : objectPlaceholder}
-        onBlur={onFieldUpdate(field.id, path)}
-        icon={<XIcon className="text-muted-foreground" />}
-        onIconClick={onRemoveField(field.id, path)}
-        disabled={arrayChild && type === 'object'}
-      />
-      {child.length > 0 ? (
-        <VCol variant="md" className="w-full border-l border-dashed">
-          {child.map((field) => (
-            <FormRenderer
-              key={field.id}
-              field={field}
-              path={childPath}
-              arrayChild={type === 'array'}
-              onAddField={onAddField}
-              onFieldUpdate={onFieldUpdate}
-              onRemoveField={onRemoveField}
+      <HCol className="w-full">
```

Comment:
Actually, I have a component that is named `Row`, becasue the `HCol` (HorizontalCol) did not make much sense, but I did not remove it 💀